test/fw/bin/pbs_sys_report

```diff
@@ -64,36 +55,23 @@ def write_to_csv(head, csvrows, i, filepath):
 
 if __name__ == '__main__':
 
-    if len(sys.argv) < 2:
-        usage()
-        sys.exit(1)
-
-    try:
-        opts, args = getopt.getopt(sys.argv[2:], "h",
-                                   ["help", "output-file="])
-    except getopt.GetoptError as err:
-        print(err)
-        usage()
-        sys.exit(1)
-
-    with open(sys.argv[1]) as fp:
-        td = json.load(fp)
-
-    filepath = None
-    for o, val in opts:
-        if o in ("-h", "--help"):
-            usage()
-            sys.exit(0)
-        elif o == "--output-file":
-            filepath = val
+    msg = 'System monitoring results to csv conversion tool'
+    parser = argparse.ArgumentParser(description=msg)
+    parser.add_argument('PTL_JSON_FILE', help='path to ptl_test_results.json')
+    parser.add_argument('--outputfile', nargs='?',
+                        help='path to generate file')
```

Comment:
This should be nargs=1, default='sys_report.csv'

Reply:
@bhroam added default, adding nargs would make it a list of 1 variable, without nargs its just a string.